src/ui/Minimap.tsx

```diff
@@ -1,20 +1,20 @@
 import { OrthographicCamera, useFBO, useTexture } from '@react-three/drei'
 import { createPortal, useFrame, useThree } from '@react-three/fiber'
 import { useEffect, useLayoutEffect, useMemo, useRef, useState } from 'react'
-import { Box3, Matrix4, Scene, Vector3 } from 'three'
+import { Box3, Matrix4, Mesh, Object3D, Scene, Vector3, WebGLRenderTarget } from 'three'
 import { useStore, levelLayer } from '../store'
 
 const m = new Matrix4()
 const v = new Vector3()
 
-function useLevelGeometricProperties() {
+function useLevelGeometricProperties(): [Box3, Vector3, Vector3] {
   const [box] = useState(() => new Box3())
   const [center] = useState(() => new Vector3())
   const [dimensions] = useState(() => new Vector3())
-  const level = useStore((state) => state.level)
+  const level = useStore((state) => state.level) as React.MutableRefObject<Scene>
```

Comment:
@drcmda Is `level` being used as Scene or Object3D I couldn't find any reference to this.